.flake8

```diff
@@ -96,6 +96,7 @@ per-file-ignores =
   # The following ignores must be fixed and the entries removed from this config:
   src/awx_plugins/credentials/aim.py: ANN003, ANN201, B950, CCR001, D100, D103, LN001, Q003, WPS210, WPS221, WPS223, WPS226, WPS231, WPS336, WPS432
   src/awx_plugins/credentials/aws_secretsmanager.py: ANN003, ANN201, D100, D103, WPS111, WPS210, WPS226, WPS329, WPS529
+  src/awx_plugins/credentials/aws_assumerole.py: WPS226, WPS332
```

Comment:
I've disabled WPS332 globally and tweaked WPS226 to have a higher threshold. So this entry is no longer necessary.
```suggestion
```